lib/hexpm/web/controllers/user_controller.ex

```diff
@@ -2,7 +2,7 @@ defmodule Hexpm.Web.UserController do
   use Hexpm.Web, :controller
 
   def show(conn, %{"username" => username}) do
-    if user = Users.get_by_username(username, [:owned_packages, :emails]) do
+    if user = Users.get_by_username(username, [:emails, owned_packages: :repository]) do
       packages = Packages.attach_versions(user.owned_packages) |> Enum.sort_by(& &1.name)
```

Comment:
![zrzut ekranu 2017-07-29 o 11 17 28](https://user-images.githubusercontent.com/76071/28743998-eb93adc8-7457-11e7-9f35-5f5603d14bae.png)

In the future we can do: `|> Enum.sort_by(&{&1.repository_id, &1.name})` to group them, I don't know yet which way I like more 😅